source/develop/release-management/features/network/autodefine-external-network.md

```diff
@@ -0,0 +1,134 @@
+  remove all dependent external networks with physical one. 
+
+* The switch type of existing clusters cannot be changed (from Linux bridge to 
+  OVS or vice versa). Thus, we currently do not need to worry about 
+  auto-defining external networks on top of existing physical networks.
+
+* Currently updating the physical network won't update auto-defined external 
+  network. This might change in the future. 
+
+* VMs will be scheduled only on hosts that does have attached linked physical 
+  network to one of it's interfaces. Same role applies for hotplug of VM 
+  interfaces. 
+
+## Detailed description
+
+### Creating of new physical network
+
+When creating new network choose its name. Check the `VM network`. If vlan 
+tagging is needed, check the `Enable VLAN tagging` and put the tag into the 
+field. Please note that auto-define can be triggered only on top of network that
+is not marked as external. 
+
+![Creation of physical network](../../../../images/wiki/auto-define_create.png  "New physical network")
+
+After confirmation if all conditions were met, the external network connected to 
+the physical one, will be created simultaneously. The auto-defined external 
+network is named `external_name_of_physical_network` (The physical network name
+might be shortened so that auto-defined network name is not bigger than 256 
+characters). 
+
+![Auto-define network created](../../../../images/wiki/auto-define_created.png  "Auto-define external network")
+
+In case there are clusters with different default external providers in the data 
+center the Engine will create multiple external networks for every provider and 
+add a number behind to this auto-defined network name. 
+
+Please note that all hosts running VMs connected to the external network has
+to be attached to the physical network. 
+
+
+
+## Implementation
+
+### Engine
+
+* A new query `GetAllVmNetworksByClusterId` that returns only VM networks is 
+  added.For Cluster with OVS switch type, this query returns only external 
+  networks. 
+
+* Validations for all limited operations with auto-defined external network and 
+  its linked physical network. According to [Autodefine External Network - Limitations](#limitations)
+
+* Added new command `AutodefineExternalNetwork`. This command is 
+  triggered every time a new network is created. The command validates the 
+  incoming network and its parameters. Then it finds all clusters with OVS 
+  switch type attached to incoming network and clusters external providers.
+  The command continues with creating new external network and connecting it to 
+  the incoming physical network and attaching to clusters. The command might 
+  create multiple external networks, one for each external provider in the data 
+  center. 
+  
+* Added new function `getAllExternalNetworksLinkedToPhysicalNetwork` into 
+  `NetworkDao` which returns list of all external networks linked to provided
+  physical network. 
+
+
```

Comment:
Is here a paragraph for the new scheduling required?

Reply:
I don't think so